ui/src/pages/frontsite/components/Series/SeriesLiteratureSearch/SeriesLiteratureSearchResultsGrid.js

```diff
@@ -0,0 +1,44 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { BookCard } from '@pages/frontsite/components/BookCard';
+import { Card, Responsive } from 'semantic-ui-react';
+import { ResultsGrid } from 'react-searchkit';
+import { SeriesCard, findVolume } from '@pages/frontsite/components/Series';
+import { recordToPidType } from '@api/utils';
+import get from 'lodash/get';
+
+export default class SeriesLiteratureSearchResultsGrid extends Component {
+  renderResultsGrid = results => {
+    // TODO: Change to use the pid object's pid_type when it's been implemented
```

Comment:
can you be more explicit with this comment? I am not sure I understand what it means.